static/src/stylesheets/module/site-messages/_engagement-banner.scss

```diff
@@ -8,12 +8,10 @@
     }
 
     @mixin padding {
-        padding-top: 9px;
-        padding-bottom: 18px;
+        padding: 9px 0;
 
         @include mq(tablet) {
-            padding-left: $gs-gutter;
-            padding-right: $gs-gutter;
+            padding: 9px $gs-gutter 18px;
```

Comment:
Can we change 9 to `$gs-baseline/1.5` and 18 to `$gs-baseline*1.5`? 

this will chop off a pixel (`baseline / 1.5' is 8, not 9. THe other one is still 18) (which makes the banner even smaller! woo!!) _and_ makes the code look so much neater on the grid system css